Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
     "vm",
     "hint_accountant",
     "./deps/parse-hyperlinks",
+    "./custom_hint_example"
```

Comment:
I was thinking it could be added as an "example" instead of as a crate. That way it shouldn't be considered by codecov.

Reply:
For that, I need to change the folder structure to put the example inside an `examples` folder (https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples), is it fine?

Reply:
I think that would be for the better

Reply:
I am not allowed to add examples in root `Cargo.toml`, because it is a workspace definition.
I added the example definition, in the vm package. I could have also kept the same folder structure for the example, but changed it anyway to the folder `examples`, because it is a rust standard and it seems cleaner.
Plz check if it is the intended solution

Reply:
I give my 👍 to that solution. WDYT @MegaRedHand?

Reply:
I think the same. This way it's easier to find them in the repo.